toolchains/xslt-M4/document/xml/element-reference-html.xsl

```diff
@@ -49,19 +49,32 @@
          <xsl:apply-templates/>
        </div>
    </xsl:template>
-   
+   <!-- The Ancor-Template Generation that supposedly slows down the page response (for ALL MODLES especially) -->
    <xsl:template match="*[exists(@gi)]" expand-text="true">
       <xsl:variable name="level" select="count(ancestor-or-self::*[exists(@gi)])"/>
       <xsl:variable name="header-tag" select="if ($level le 6) then ('h' || $level) else 'p'"/>
       <div class="model-entry definition { tokenize(@_metaschema-xml-id,'/')[2] }">
          <xsl:variable name="header-class" expand-text="true">{ if (exists(parent::map)) then 'definition' else 'instance' }-header</xsl:variable>
          <div class="{ $header-class }">
-            <!-- generates h1-hx headers picked up by Hugo toc -->
-            <xsl:element expand-text="true" name="{ $header-tag }" namespace="http://www.w3.org/1999/xhtml">
-               <xsl:attribute name="id" select="@_tree-xml-id"/>
-               <xsl:attribute name="class">toc{ $level} name</xsl:attribute>
-               <xsl:text>{ @gi }</xsl:text>
+
+            <!-- generates h1-hx headers picked up by Hugo ToC -->
+
+            <!-- ===!!! Anchor hard wrap-around of Headers 1-6 [see logic above with ($level le 6)] !!!=== -->               
+            <xsl:element expand-text="true" name="a" namespace="http://www.w3.org/1999/xhtml">
+
+               <xsl:attribute name="href">#{@_tree-xml-id}</xsl:attribute>
+               <xsl:attribute name="class">no-anchor-xslt toc{$level} name</xsl:attribute>
+               <xsl:attribute name="title">Focus on {@gi} details</xsl:attribute>
+
+                  <!-- ===!!! The Headers 1-6 that are being wrapped around !!!=== -->
+               <xsl:element expand-text="true" name="{ $header-tag }" namespace="http://www.w3.org/1999/xhtml">
+                  <xsl:attribute name="id" select="@_tree-xml-id"/>
+                  <xsl:attribute name="class">no-anchor-xslt toc{ $level} name Elem-Ref-Html--Xsl-72</xsl:attribute>
```

Comment:
Can we adjust the the class name per the feedback provided in parallel with https://github.com/usnistgov/OSCAL/pull/1789#pullrequestreview-1428721803?

```suggestion
                  <xsl:attribute name="class">reference-element-anchor toc{ $level} name</xsl:attribute>
```